_sass/leaderboard.scss

```
@@ -29,6 +29,31 @@
   }
 }
 
+.leaderboard.dark {
+  
+  a {
+    color: white;
+  }
+
+  tr:nth-child(2n + 1) {
+    color: white;
+  }
+
+  tr:nth-child(2n) {
+    background-color: grey;
```

While it's easy to use the named colors (e.g. red, blue, grey ...) they are often a little too general to fit nicely. I'd recommend something like: 

```suggestion
    background-color: #23252B;
```